src/test/rustdoc-gui/mobile.goml

```diff
@@ -0,0 +1,16 @@
+goto: file://|DOC_PATH|/staged_api/struct.Foo.html
```

Comment:
Please add a little description of what this is testing (I should maybe enforce it with a lint?).

Reply:
Done.